src/errors.rs

```diff
@@ -24,6 +25,9 @@ pub enum DkgError {
     #[error("point error {0:?}")]
     /// An error during point operations
     Point(PointError),
+    #[error("try_from failed")]
+    /// An error during try_from operations
+    TryFrom,
```

Comment:
This is okay, but why not be more specific and call it `TryFromInt`? My thinking is that you may want to have another error when converting from some other type besides integers.

Reply:
I originally did call it TryFromInt, and embedded the error. But that caused problems with some trait bounds, so I decided to just make a single TryFrom without any embed. 

Realistically we’re only ever gonna convert between integers so whatever’s clever.

Reply:
Ok, changed it to `TryFromInt`

Reply:
🙏🏽